nuxt.config.js

```diff
@@ -10,7 +10,9 @@ export default {
     router: {
         base: process.env.ROUTER_BASE || DEFAULT_ROUTER_BASE,
     },
-
+    server: {
```

Comment:
why do we need these lines?

Reply:
Oops I pushed the workaround script for mobile phone layout checking. I will fix it next commit also update to use tailwind css styling.